### PR TITLE
Add operations/faqs docs and layout change

### PR DESCRIPTION
**Pull request by cofyc**
- operations and faqs docs
  - docs/operations.md
  - docs/faqs.md
- simplify README.md
  - user guide -> docs/getting-started.md
  - best praticces -> docs/best-practices.md
- add TOC to help readers navigate on the website

https://github.com/cofyc/sig-storage-local-static-provisioner/tree/setup-docs

### CODE REVIEW

File: TODO.md [hunk 1]
```diff
@@ -6,7 +6,6 @@
 ## P1
 
 ## P2
-* Dynamic provisioning using lvm
```
Comment (cofyc) [inline TODO.md]: deleted because it is not in the plan of this project

File: README.md [hunk 2]
```diff
@@ -290,38 +132,6 @@ Run `./hack/e2e.sh -h` to view help.
...
-  mistaken for a volume on another node with the same name.
-* For raw block volumes without a filesystem, use a unique ID as the symlink
-  name. Depending on your environment, the volume's ID in `/dev/disk/by-id/`
-  may contain a unique hardware serial number. Otherwise, a unique ID should be 
-  generated. The uniqueness of the symlink name will ensure that if another 
-  node with the same name is created, that any volumes on that node are 
-  unique and not mistaken for a volume on another node with the same name.
-
-
-### Deleting/removing the underlying volume
```
Comment (cofyc) [inline README.md]: This section is merged into operation docs.

File: helm/README.md [hunk 3]
```diff
@@ -53,12 +53,13 @@ provisioner chart and their default values.
...
 | common.useJobForCleaning                     | If set to true, provisioner will use jobs-based block cleaning.                                       | bool     | `false`                                                    |
 | common.useNodeNameOnly                       | If set to true, provisioner name will only use Node.Name and not Node.UID.                            | bool     | `false`                                                    |
 | common.minResyncPeriod                       | Resync period in reflectors will be random between `minResyncPeriod` and `2*minResyncPeriod`.         | str      | `5m0s`                                                     |
 | common.configMapName                         | Provisioner ConfigMap name.                                                                           | str      | `local-provisioner-config`                                 |
+| common.podSecurityPolicy                     | Whether to create pod security policy or not.                                                         | bool     | `false`                                                    |
```
Comment (cofyc) [inline helm/README.md]: https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/pull/73

File: helm/README.md [hunk 4]
```diff
@@ -53,12 +53,13 @@ provisioner chart and their default values.
...
 | common.rbac                                  | Generating RBAC (Role Based Access Control) objects.                                                  | bool     | `true`                                                     |
 | common.namespace                             | Namespace where provisioner runs.                                                                     | str      | `default`                                                  |
-| common.createNamespace                       | Whether to create namespace for provisioner.                                                          | bool      | `false`                                                  |
+| common.createNamespace                       | Whether to create namespace for provisioner.                                                          | bool     | `false`                                                    |
 | common.useAlphaAPI                           | If running against pre-1.10 k8s version, the `useAlphaAPI` flag must be enabled.                      | bool     | `false`                                                    |
```
Comment (cofyc) [inline helm/README.md]: removed an extra space

**Comment by msau42** [comment]
This is really awesome! Thanks!
/lgtm

**Pull request merged**